local_flow/intent/src/cart_baseline_flow.py

```diff
@@ -123,38 +123,41 @@ def get_model_config(self):
     @enable_decorator(batch(gpu=1, cpu=8, image=os.getenv('BASE_IMAGE')),
                       flag=os.getenv('EN_BATCH'))
     # @ environment decorator used to pass environment variables to Batch instance
-    @environment(vars={'WANDB_API_KEY': os.getenv('WANDB_API_KEY'),
+    @environment(vars={'NEPTUNE_PROJECT': os.getenv('NEPTUNE_PROJECT'),
+                       'NEPTUNE_API_TOKEN': os.getenv('NEPTUNE_API_TOKEN'),
+                       'NEPTUNE_CUSTOM_RUN_ID': os.getenv('NEPTUNE_CUSTOM_RUN_ID'),
+                       'WANDB_API_KEY': os.getenv('WANDB_API_KEY'),
                        'WANDB_ENTITY': os.getenv('WANDB_ENTITY'),
                        'BASE_IMAGE': os.getenv('BASE_IMAGE'),
                        'EN_BATCH': os.getenv('EN_BATCH')})
-    # @ custom pip decorator for pip installation on Batch instance
-    @pip(libraries={'wandb': '0.10.30'})
+    # un-comment if provided image does not contain required packages
+    # @pip(libraries={'wandb': '0.10.30', "neptune-client": "0.13.3", "neptune-tensorflow-keras": "0.9.9"})
```

Comment:
It would definitely be a better approach. It saves the user a lot of trouble. 

Quick question. How do I define the minimum version of a package using this decorator `@pip(...)`?

Something like `neptune-client>=0.13.3` but inside this decorator

Reply:
@lgarciaCoveo is the above possible? Currently, the neptune-client is being constantly updated. So users can have the latest and greatest.  Our integration (neptune-tensorflow-keras) package can have a fixed version.

Reply:
I think I got it.
In your [custom decorator page](https://github.com/jacopotagliabue/you-dont-need-a-bigger-boat/blob/main/local_flow/intent/src/custom_decorators.py) you specify if the version is empty it will install the latest version.

Reply:
Optionally, we can talk about potential changes to the custom decorator to support a minimal version.